tests/dtypes/test_geometry.py

```diff
@@ -135,3 +145,153 @@ def test_geometry_params_srid(
             'coordinates': 'POINT (582710 6061887)',
         }
     ]
+
+
+def test_geometry_html(rc: RawConfig):
+    context, manifest = load_manifest_and_context(rc, '''
+    d | r | b | m | property                   | type           | ref
+    example                                    |                |
+      |   |   | City                           |                |
+      |   |   |   | name                       | string         |
+      |   |   |   | coordinates                | geometry(4326) |
+    ''')
+    result = render_data(
+        context, manifest,
+        'example/City',
+        None,
+        accept='text/html',
+        data={
+            '_id': '19e4f199-93c5-40e5-b04e-a575e81ac373',
+            '_revision': 'b6197bb7-3592-4cdb-a61c-5a618f44950c',
+            'name': 'Vilnius',
+            'coordinates': 'POLYGON',
+        },
+    )
+    assert result == {
+        '_id': Cell(
+            value='19e4f199',
+            link='/example/City/19e4f199-93c5-40e5-b04e-a575e81ac373',
+            color=None,
+        ),
+        'name': Cell(
+            value='Vilnius',
+        ),
+        'coordinates': Cell(
+            value='POLYGON',
+            link=None,
+            color=None,
+        ),
+    }
+
+
+@pytest.mark.parametrize('value, cell', [
+    (
+        {
+            'geom_type': 'Point',
+            'geom_value': [582710, 6061887],
+            'srid': WGS84
+        },
+        Cell(value='POINT (582710 6061887)',
+             link='https://www.openstreetmap.org/?mlat=582710.0&mlon=6061887.0#map=19/582710.0/6061887.0')
+    ),
+    (
+        {
+            'geom_type': 'Point',
+            'geom_value': [582710, 6061887],
+            'srid': LKS94
+        },
+        Cell(value='POINT (25.282777879597916 54.68661318326901)',
+             link='https://www.openstreetmap.org/?mlat=25.282777879597916&mlon=54.68661318326901'
+                  '#map=19/25.282777879597916/54.68661318326901')
+    ),
+    (
+        {
+            'geom_type': 'Polygon',
+            'geom_value': [[[502640, 6035290], [502630, 6035290], [502630, 6035300]]],
+            'srid': WGS84
+        },
+        Cell(value='POLYGON',
+             link='https://www.openstreetmap.org/?mlat=502633.3333333334&mlon=6035293.333333334'
+                  '#map=19/502633.3333333334/6035293.333333334')
+    ),
+    (
+        {
+            'geom_type': 'Polygon',
+            'geom_value': [[[502640, 6035290], [502630, 6035290], [502630, 6035300]]],
+            'srid': LKS94
+        },
+        Cell(value='POLYGON',
+             link='https://www.openstreetmap.org/?mlat=24.040608716271&mlon=54.454444306004426'
+                  '#map=19/24.040608716271/54.454444306004426')
```

Comment:
I think, it is enough to just add POINT for this test, in order to test transformation, probably tests with POLYGON is redundant?